justfile

```
@@ -71,10 +71,10 @@ setup-cluster: create-cluster
   kubectl apply -f https://raw.githubusercontent.com/metallb/metallb/v0.13.9/config/manifests/metallb-native.yaml
   kubectl wait -n metallb-system --for=condition=ready pod -l app=metallb --timeout=240s
   kubectl apply -f https://kind.sigs.k8s.io/examples/loadbalancer/metallb-config.yaml
-  kubectl apply -f https://raw.githubusercontent.com/traefik/traefik/v2.6/docs/content/reference/dynamic-configuration/traefik.containo.us_ingressroutes.yaml
+  kubectl apply -f https://raw.githubusercontent.com/traefik/traefik/v2.10/docs/content/reference/dynamic-configuration/traefik.io_ingressroutes.yaml
   kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.11.0/cert-manager.yaml
   helm repo add bitnami https://charts.bitnami.com/bitnami
-  helm upgrade --install --set crd.create=true --wait external-dns bitnami/external-dns
+  helm upgrade --install --set crd.create=true --wait external-dns bitnami/external-dns --debug --timeout 10m
```

Are these two flags necessary?

```suggestion
  helm upgrade --install --set crd.create=true --wait external-dns bitnami/external-dns
```

---

We can remove the debug flag, but I think that the timeout option will be required for this to work